packages/ListBoxBrowser/README.md

```diff
@@ -0,0 +1,173 @@
+
+Also, has two functions to set defaults for selected options and a default selected view for the browser column.
+
+The `onChange` prop receives a parameter with an array of the selected options.
+
+### Data prop
+
+The most important prop for the `<ListBoxBrowser />` is the `data` prop, which initializes the state for the component.
+
+The `data` prop has a shape of an array of objects where each object requires to have at least a `label` attribute on it, and at least one of those items should include an `options` property.
+
+### Data prop shape
+
+```js
+{
+	label: "require attribute",
+	options: [{}{}{}], // more object with the same shape
+}
+```
+
+The **object** can have an options attribute, which is an array of the same kind of objects.
+
+A data prop with **options** looks like:
+
+```js
+const data = [
+   {
+        label: "NBA",
+        options: [{ label: "Toronto Raptors"}, {...}]
+   },
+   {
+        label: "NHL",
+        options: [{ label: "Vancouver Canucks"}, {...}]
+   },
+   {
+        label: "MLS",
+        options: [{ label: "Vancouver Whitecaps"}, {...}]
+   },
+]
+```
+
+Recursively the component runs over all the options on the array and assembles the appropriated UI for the user.
+
+**NOTE** You can add extra properties to the objects; those are and pass to you back on the `onChange` function or when using `defaultSelectedOptions`, `defaultSelectedView` functions.
+
+---
+
+#### Basic example
+
+```js
+import ListBoxBrowser from "@paprika/listbox-browser";
+
+export default function App() {
+   const data = [{...}...{...}]; // your data;
+   return <ListBoxBrowser data={data} />
+}
+```
+
+---
+
+#### ListBoxBrowser.SelectedOptions
+
+By default, the `<ListBoxBrowser />` does not display a list of the selected options.  
+But you can activate this feature to display multiple options using the `<ListBoxBrowser.SelectedOptions />` subcomponent.
+
+```js
+<ListboxBrowser data={data}>
+   <ListBoxBrowser.SelectedOptions />
+</ListBoxBrowser>
+```
+
+#### defaultSelectedOptions
+
+The `defaultSelectedOptions` received a function to be executed to determine if an option should start as selected or not. This function is called as many times as options are pass down on the data prop.
+
+Ex.
+
+```js
+const data = [
+	{ key: 1, label: "one", options: [{...}]},
+	{ key: 2, label: "two", options: [{...}]},
+	...
+]
+
+<ListBoxBrowser
+	data={data}
+	defaultSelectedView={(option) =>  (option.key === 1)}  />
+
+/**
+  The previous result on option with `key 1` to be selected */
+```
+
+#### defaultSelectedView
+
+This prop worked exactly like the previous one, receiving a function which you can use to compare and decide what option should be the one as initial view.
+
+**Note**: Be sure to selected an `option` property with `options`; otherwise, this function doesn't work and fall back to the default behaviour, which is to select the first option with options.
```

Comment:
```suggestion
**Note**: Be sure to select an `option` property with `options`. Otherwise, this function doesn't work and falls back to the default behaviour, which selects the first option with options.
```